lesson04/assignment/css/_layouts.scss

```diff
@@ -0,0 +1,201 @@
+@mixin grid-header {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: repeat(3, 1fr);
+    width: 100%;
+    height: $xxxl-rem;
+ }
+
+// Header nav-bar
+@mixin grid-nav-list {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    justify-items: center;
+    list-style-type: none;
+    @include mrgn-pdng-set-0;
+    height: $xxl-rem;
+}
+
+// Grid article (section)
+@mixin grid-article {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: 2fr, 1fr;
+    align-items: center;
+    @include full-x-fit-y;
+    background: transparent;
+}
+
+// Specific grids
+@mixin col1-row1 {
+    grid-column: 1;
+    grid-row: 1;
+}
+
+@mixin col1-row2 {
+    grid-column: 1;
+    grid-row: 2;
+}
+
+@mixin col2-row1 {
+    grid-column: 2;
+    grid-row: 1;
+}
+
+@mixin col2-row2 {
+    grid-column: 2;
+    grid-row: 2;
+}
+
+@mixin col1_3-row2 {
+    grid-column: 1 / 3;
+    grid-row: 2;
+}
+
+
+// Flexbox article (section)
+@mixin fb-article {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: transparent;
+    @include full-x-fit-y;
+}
+
+// Flexbox footer
+@mixin fb-footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    @include full-x-fit-y;
+    background-image: url(../img/kyler-boone-LA-skies-unsplash.jpg);
+    background-position: top;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+// Custom styling
+
+// Quotes common styling
+@mixin common-quote-styles {
+    font-family: $quotes-font;
+    color: $platinum;
+    font-size: $l-font;
+    letter-spacing: $xs-pxl;
+    line-height: $xl-rem;
+    margin: 0 auto;
+    margin-top: $l-rem;
+    margin-bottom: $l-rem;
+}
+
+// Author credit quote common styling
+@mixin common-quote-font {
+    font-family: $small-ital-font;
+    font-size: $s-font;
+    font-style: italic;
+}
+
+// Paragraph Article common styling
+@mixin common-text-article {
+    width: 60%;
+    margin: 0 auto;
+    font-family: $p-font;
+    letter-spacing: $xs-pxl;
+    line-height: $m-rem;
+}
+
+// Buttons Article common styling
+@mixin common-btn-article {
+    font-family: $btn-font;
+    font-size: $xs-font;
+    background: transparent;
+    width: fit-content;
+    justify-self: center;
+    margin-bottom: $m-rem;
+    padding: $xxs-rem $xs-rem;
+    border-radius: $s-pxl;
+    cursor: pointer;
+}
+
+// Logo 
+@mixin logo-item {
+    font-family: $logo-btn-font;
+    text-decoration: none;
+    color: $white;
+    // transition: all 1s;
+    cursor: pointer;
+    letter-spacing: $xl-pxl;
+    align-self: center;
+    text-shadow: $xxxxs-rem $xxxxs-rem 0 $shadow-logo;
+    transition: all 1s ease-in-out;
+}
+
+// Anchor tags Nav Bar styling
+@mixin navbar-item {
+    text-decoration: none;
+    color: $white;
+    font-size: $xs-font;
+    transition: all .5s ease-in-out;
+    cursor: pointer;
+}
+
+// Paralax common styling
+@mixin parallax {

```

Comment:
Great work with the scrolling, looks awesome

Reply:
Thanks!!!